crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -336,7 +336,7 @@ impl RefreshTask {
                     if let Some(batch) = stream.next().await {
                         match batch {
                             Ok(batch) => {
-                                tracing::info!(
+                                tracing::trace!(
```

Comment:
I think its useful to have this trace (as basically a version of progress loading that we wanted), but probably tone it down to it doesn't spam so much (i.e. accumulate the number of rows within a 10 second window and only emit a lot once every 10s).

Reply:
Thank you, accumulated tracing is a great idea, I'm going to merge this and implement it. Tracing on batch level is still useful for debugging / troubleshooting IMO